src/adams_utils.jl

```diff
@@ -3,8 +3,15 @@ function ϕ_and_ϕstar!(cache, dy, next_point, last_idx)
   for i = 0:(k)-1
     if i == 0
       β[(i)+1] = 1
-      ϕ_n[(i)+1] = copy(dy)
-      ϕstar_n[(i)+1] = copy(dy)
+      if typeof(dy) <: Array
+        # ϕ_n[(i)+1] = copy(dy)
+        ϕ_n[(i)+1] .= dy
+        # ϕstar_n[(i)+1] .= dy
```

Comment:
@ChrisRackauckas Why broadcasting is not working (convergence tests are failing) here. I could not figure out what is the problem.

Reply:
:thinking:

Reply:
so it passes when it's a copy?

Reply:
Are all of the `ϕstar_n` vectors initialized in a way that they alias?

Reply:
yeah